### PR TITLE
adding photo gallery

### PR DESCRIPTION
**Pull request by shreyabarua**
adding slideshow photo gallery to portfolio

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -112,7 +112,42 @@ <h2 id = "cooking" > Cooking </h2>
...
+            <img class = "slide" src="/images/littlepecks.jpg">
+        </div>
+
+        <div class = "mySlides">
+            <img class = "slide" src="/images/group3.jpg">
+        </div>
+
+        <a class = "prev" onclick="plusSlides(-1)">❮</a>
+        <a class = "next" onclick="plusSlides(1)">❯</a>
+
```
Comment (oydang) [inline portfolio/src/main/webapp/index.html]: These lines should be indented once more.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,6 +12,20 @@
...
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
+  for (i = 0; i < slides.length; i++) {
+      slides[i].style.display = "none";
+  }
+  slides[slideIndex-1].style.display = "block";
+}
```
Comment (oydang) [inline portfolio/src/main/webapp/script.js]: Nit: newline @ end of file

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -135,4 +135,54 @@
...
+.next {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover,
+.next:hover {
+  background-color: rgba(0, 0, 0, 0.8);
 }
```
Comment (oydang) [inline portfolio/src/main/webapp/style.css]: Nit: newline at end of the file

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,6 +12,20 @@
...
+
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
```
Comment (oydang) [inline portfolio/src/main/webapp/script.js]: Nit: don't need curly braces for 1-line if-statements.

https://google.github.io/styleguide/jsguide.html#formatting-braces-all

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -12,6 +12,20 @@
...
-*/
+var slideIndex = 1;
+showSlides(slideIndex);
+
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var i;
```
Comment (oydang) [inline portfolio/src/main/webapp/script.js:23]: This variable isn't used anywhere.

**Review by oydang — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (shreyabarua) [reply portfolio/src/main/webapp/script.js:23]: var i is used in the for loop below

Comment (oydang) [reply portfolio/src/main/webapp/script.js:23]: My bad, didn't see that!

**Review by oydang — APPROVED** [review]

**Pull request merged**